app/assets/javascripts/components/on-boarding/terminal-window.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jsx React.DOM
+ */
+
+var React = require('react');
+
+//React components
+var TerminalPrompt = require('./terminal-prompt');
+
+var TerminalWindow = React.createClass({
+  getInitialState: function(){
+    return{
+      message: "Enter your email address to sign up."
+    }
+  },
+  handleKeyDown: function(e){
+    //Setup listener for key events in order
+    //to preceed the text with blinking cursor
+    switch (e.keyCode) {
+      case 8:  // Backspace
+        console.log('backspace');
+        break;
+      case 9:  // Tab
+      case 13: // Enter
+      case 37: // Left
+      case 38: // Up
+      case 39: // Right
+      case 40: // Down
+        break;
+      default:
+        this.moveCursor()
+    }
+  },
+  update: function(e){
+    //TODO: Need real live validation here for emails
+    if(e.target.value === 'ed@htmelvis.com'){
+      var msg = "What's up Ed!";
+    } else{
+      var msg = "OOOPpppss you seem to have not typed in an email address";
+    }
+
+    this.setState({
+      message: msg
```

Comment:
For naming purposes and understanding, would it make sense to rename this property to `terminalMessage` ?